### ISSUE TITLE
[11.0] account_invoice_validation_queued : odoo.modules.registry: At least one model cache has been invalidated

### ISSUE THREAD

**Issue by qupro** [OCA/account-invoicing#703]
When I copy "account_invoice_validation_queued" module in  "/odoo/custom/addons" path I get "INFO test odoo.modules.registry: At least one model cache has been invalidated, signaling through the database." message. 

Also, at the web interface in localhost:8069/web/database/manager, no css is loaded.

**Comment by pedrobaeza** [comment]
The message about invalidation is correct and normal to happen. The other has nothing to do with that module, although maybe is motivated by incorrect permissions in your file system.

Any way, it's a deployment question and not a problem in the module itself, so closing this issue.

**Comment by qupro** [comment]
Thanks for your quick reply, 

at the beginning I also thought it could be a file permission problem, but I copy this module along with the rest of account-invoicing modules in addons path and all of them have the same permissions (odoo:odoo 775). I have just re-check it.

After I got the error, I started to move modules  in and out  from addons path until I found that "account_invoice_validation_queued" module was the only one that produce this error.
